userspace/libsinsp/test/dns_manager.ut.cpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#if defined(HAS_CAPTURE) && !defined(CYGWING_AGENT) && !defined(_WIN32) && !defined(__EMSCRIPTEN__)
+#include "sinsp_with_test_input.h"
+#include <dns_manager.h>
+#include <gtest/gtest.h>
+
+TEST_F(sinsp_with_test_input, simple_dns_manager_invocation) 
+{
+    // Simple dummy test to assert that sinsp_dns_manager is invocated correctly
+    // and not leaking memory
```

Comment:
Do we run the test suite under valgrind in CI? Do we actually detect the (lack of) memory leak?